xla/service/cpu/onednn_contraction_rewriter.h

```diff
@@ -1,4 +1,4 @@
-/* Copyright 2023 The OpenXLA Authors.
+/* Copyright 2024 The OpenXLA Authors.
```

Comment:
No need to update the year. It should be the year the file was created. Same for other files.
```suggestion
/* Copyright 2023 The OpenXLA Authors.
```